main.js

```diff
@@ -32,3 +32,53 @@ function start() {
   }
 }
 start();
+var sundayElement = document.getElementsByClassName("day")[0];
+sundayElement.addEventListener("click", sundaySchedule);
+var mondayElement = document.getElementsByClassName("day")[1];
+mondayElement.addEventListener("click", mondaySchedule);
+var tuesdayElement = document.getElementsByClassName("day")[2];
+tuesdayElement.addEventListener("click", tuesdaySchedule);
+var wednesdayElement = document.getElementsByClassName("day")[3]
+wednesdayElement.addEventListener("click", wednesdaySchedule);
+var thursdayElement = document.getElementsByClassName("day")[4];
+thursdayElement.addEventListener("click", thursdaySchedule);
+var fridayElement = document.getElementsByClassName("day")[5];
+fridayElement.addEventListener("click", fridaySchedule);
+var saturdayElement = document.getElementsByClassName("day")[6];
+saturdayElement.addEventListener("click", saturdaySchedule);
```

Comment:
Using event bubbling you can cut this down to one listener
and don't use getElementsByClassName use querySelector or querySelectorAll

And also if you are going to use querySelectorAll you can grab the values from the nodelist rather than calling the same function 6 times